### DISCUSSION TITLE
Filament load and unload in start and end macros

### DISCUSSION THREAD

**Discussion by mrhasbean** [jschuh/klipper-macros#99] [Q&A]
> 'The load_length variable is used as a default value by the [filament macros](https://github.com/jschuh/klipper-macros#filament). It's not used in PRINT_START or PRINT_END.

Oh ok, so the macros assume that filament will be loaded and unloaded manually in a separate step? That's where my confusion was, I've just never seen it done that way before. Thanks for taking the time to reply.

**Comment by jschuh** [comment]
No problem.
